docs/guides/account-user-mgmt/profile-management.md

```diff
@@ -0,0 +1,36 @@
+---
+valeOff: <!-- vale off -->
+heading: Manage Accounts and Users
+seo: Account APIs Overview | Cloud Elements API Docs
+title: Manage Your Profile
+description: Find API documentation, account management guides, and more on all of the currently supported Account APIs.
+layout: sidebarleft
+platform: accounts
+breadcrumbs: /docs/guides/home.html
+parent: Back to Guides
+order: 30
+redirect_from:
+  - /docs/platform/accounts/account-management.html
+ValeOn: <!-- vale on -->
+---
+
+# Manage Your Profile
+
+Your profile includes the basic information about yourself, like your name and email. It also includes critical information that you need to interact with Cloud Elements including your password, User Secret, and Organization secret.
```

Comment:
If you capitalized "User *S*ecret", should you capitalize the "Organization *s*ecret" as well?